src/hooks/useStorage/index.ts

```diff
@@ -0,0 +1,4 @@
+import useLocalStorage from './useLocalStorage';
+import useSessionStorage from './useSessionStorage';
+
+export { useLocalStorage, useSessionStorage };
```

Comment:
useLocalStorage, useSessionStorage 따로 별도의 파일로 처리하는 것보다, index.js 에서 선언하고 export 하는 것이 더 깔끔하게 보여진다고 생각하는데 준혁님, 다른 분들 의견 어떤 지 궁금합니다.
```suggestion
import useStorage, { ReturnTypes } from './useStorage';

function useLocalStorage<T>(key: string, defaultValue: T): ReturnTypes<T> {
  const [storage, setStorage] = useStorage(key, defaultValue, 'localStorage');
  return [storage, setStorage];
}

function useSessionStorage<T>(key: string, defaultValue: T): ReturnTypes<T> {
  const [storage, setStorage] = useStorage(key, defaultValue, 'sessionStorage');
  return [storage, setStorage];
}

export { useLocalStorage, useSessionStorage };

```

Reply:
넵 이런 식으로 사용해도 전혀 문제가 없을 것 같습니다. 다만 주로 하나의 파일에 하나의 hook 또는 컴포넌트를 작성하는 습관이 남아있어서 그렇게 사용한 것 같아요. 저도 다른 분들의 의견을 들어보고 싶습니다. ㅎㅎ

Reply:
저는 파일분리가 좀 더 가독성이 좋다고 봅니다 🤗

Reply:
저는 이 부분 굳이 분리 안해도 된다는 의견입니다 :)